demos/ns-inputter--select/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "ns-inputter--select",
+  "version": "1.0.2",
```

I guess this should this be `1.0.0`.
```suggestion
  "version": "1.0.0",
```